Doc/User_Manual.md

```diff
@@ -14,7 +14,23 @@ The `RAWcooked` tool
 - encodes with the FLAC audio codec all audio files in the folder
 - muxes these into a Matroska container (.mkv)
 
-The filenames of the single-image files must end with a numbered sequence. `RAWcooked` will generate the regular expression (regex) to parse in the correct order all the frames in the folder.
+The filenames of the single-image files must end with a numbered sequence. `RAWcooked` will generate the regular expression (regex) to parse in the correct order all the frames in the folder. 
+
+The number sequence within the filename can be formed with leading zero padding - eg 000001.dpx, 000002.dpx... 500000.dpx - or no leading zero padding - eg 1.dpx, 2.dpx... 500000.dpx.
```

Comment:
To me the abbreviation is more readable with dots (e.g.), but English is not a language I know well. Also below.

Reply:
Agree e.g. would be clearer, thanks!